src/modules/sub-items/components/table-view/table.view.component.js

```diff
@@ -249,16 +155,16 @@ export default class TableViewComponent extends Component {
                     <td className={cellHeadClass}>
                         <input type="checkbox" checked={anyLocationSelected} onChange={this.onSelectAll} />
                     </td>
-                    <td className={`${cellHeadClass} ${cellClass}--name ${cellSortClass}`} onClick={this.sortByName}>
+                    <td className={`${cellHeadClass} ${cellClass}--name ${cellSortClass}`} onClick={this.onSortByName}>
```

Comment:
The previous one was proper. `onClick` do `sortByName` not `onSortByName`.
Also applies to `onSortByDate` and `onSortByPriority`